less/plugins/adapt-contrib-hotgraphic/hotgraphicPopup.less

```diff
@@ -0,0 +1,60 @@
+.hotgraphic-popup {
+  // --------------------------------------------------
+  // Increase toolbar padding top so the browser outline of the
+  // item title is not hidden by toolbar
+  // --------------------------------------------------
+  &__inner.has-toolbar {
+    padding-top: (@icon-size + (@icon-padding * 2)) + (@item-padding / 2);
+  }
+  // --------------------------------------------------
+
+  &__controls,
+  &__close {
+    margin: @btn-margin / 2;
+    padding: @btn-padding / 2;
+    background-color: transparent;
+    color: @notify-icon-inverted;
+    border-radius: 50%;
+
+    .no-touch &:hover {
+      background-color: @notify-icon-hover;
+      color: @notify-icon-inverted-hover;
+      .transition(background-color @duration ease-in, color @duration ease-in;);
+    }
+  }
+
+  // &__controls,
```

Comment:
Should these be deleted?

Reply:
I recommend leaving comment in as code allows different style of button.